### ISSUE TITLE
Keyboard shortcuts to switch between tabs

### ISSUE THREAD

**Issue by eli-oat** [kickscondor/fraidycat#68]
**Before you start.**

* [x] I have checked the [Rejected Features](https://github.com/kickscondor/fraidycat/wiki/Rejected-Features) page.
* [ x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**My great idea for Fraidycat is:**
Add the ability to switch between active tabs with a key command, either some modifier key + left/right arrow, or maybe assign a number to each tab start from 1?

The hiccup here being finding a modifier key that isn't eaten up by the browser/OS already.

Maybe `shift` + vi movement keys?

**Comment by kickscondor** [comment]
Brilliant! I lean toward using the same shortcuts that browsers all use.

* Ctrl+1-8 – Switch to the specified tab, counting from the left.
* Ctrl+9 – Switch to the last tab.
* Ctrl+Tab, Ctrl+PageDn – Switch to the next tab – in other words, the tab on the right.
* Ctrl+Shift+Tab, Ctrl+PageUp – Switch to the previous tab – in other words, the tab on the left.
* Ctrl+N (or Ctrl+O) – Create a new follow.
* Alt+Left Arrow, Backspace – Back.
* Alt+Right Arrow, Shift+Backspace – Forward.
* Alt+Home – Open homepage.

But, as long as there is no conflict, I could see the Vim shortcuts working just fine as well. (Which is what I would personally use as well.)

I'm very grateful for these thoughtful suggestions.

**Comment by eli-oat** [comment]
For helpful context, here are the keyboard shortcuts from a few other browser based things: 

## Miniflux
```

Keyboard shortcuts

    Go to unread = gu
    Go to bookmarks = gb
    Go to history = gh
    Go to subscriptions = gs
    Go to preferences = gp
    Update all subscriptions = r
    Download content = d
    Previous item = p or j or ◄
    Next item = n or k or ►
    Mark as read or unread = m
    Mark all unread items as read = A
    Open original link = v
    Open item = o
    Bookmark item = f
    Previous page = h
    Next page = l
    Toggle RTL mode = z
    Show help = ?
    Close help = q
    Search = /
```

## Gmail
(Too bananas to be copied and pasta'd into here -- I also can't find official docs for them, sorry for the weird link!) <https://www.webnots.com/gmail-keyboard-shortcuts/>

**Comment by kickscondor** [comment]
Yes and it's occurred to me that, since many are using Fraidycat as an extension, it'll need it's own shortcuts there anyway - so your list is probably even better. Going to let this marinate for a day.

**Comment by kickscondor** [comment]
Need a global shortcut as well, to create a new follow from the current page. (Basically an alias for #99.) Perhaps a global shortcut to open Fraidycat's home page.

**Comment by nnisen** [comment]
Would be great if it was possible to expand a line with a keyboard shortcut - the button the expand is so far away on the right. Not exactly sure how it would work (e.g. first opens the first line, subsequent open the subsequent lines OR expand all).

I would use this to scan through the line items more easily.

**Comment by sogen** [comment]
adding another vote to this
